recbole/model/general_recommender/nncf.py

```diff
@@ -0,0 +1,303 @@
+# -*- coding: utf-8 -*-
+# @Time   : 2021/1/14
+# @Author : Chengyuan Li
+# @Email  : 2017202049@ruc.edu.cn
+
+import torch
+import torch.nn as nn
+from torch.nn.init import normal_
+
+from recbole.model.abstract_recommender import GeneralRecommender
+from recbole.model.layers import MLPLayers
+from recbole.utils import InputType
+
+import numpy as np
+import networkx as nx
+import community

```

Comment:
If the two packages (networkx and community) are written here, we must declare them in requirements.
Currently, only NNCF uses them. So I suggest writing them in line134 and line137, respectively.